src/Core/Domain/Customer/Group/QueryResult/EditableCustomerGroup.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\QueryResult;
+
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupId;
+
+class EditableCustomerGroup
+{
+    /**
+     * @var GroupId
+     */
+    private $id;
+
+    /**
+     * @var string[]
+     */
+    private $name;
+
+    /**
+     * @var float
+     */
+    private $reduction;
+
+    /**
+     * @var int
+     */
+    private $priceDisplayMethod;
+
+    /**
+     * @var bool
+     */
+    private $showPrice;
+
+    /**
+     * @param GroupId $id
+     * @param string[] $name
+     * @param float $reduction
+     * @param int $priceDisplayMethod
+     * @param bool $showPrice
+     */
+    public function __construct(
+        GroupId $id,
+        array   $name,
+        float   $reduction,
+        int     $priceDisplayMethod,
+        bool    $showPrice
```

Comment:
Check other places too if there are any similar code
```suggestion
        GroupId $id,
        array $name,
        float $reduction,
        int $priceDisplayMethod,
        bool $showPrice
```
cs fixer should take care of it ideally, but in case it doesn't - I think we are not using those param alignments (its just your IDE configs that adds them and they should be removed).
I think we should adjust cs fixer for this too if it doesn't work now (in dedicate PR though).

Reply:
Fixed